tests/testthat/test-subscript-QStatisticTestingInfo.R

```diff
@@ -0,0 +1,186 @@
+        1.87282843753458, 1.13201493826764, -0.489580936488417, -1.59654044426236,
+        0.498538451671911, NaN, 4.03911418475605, 0.980772638284431,
+        2.21752864240068, -4.0894391990541, -0.14334486430662, -2.09038716182298,
+        1.84247691606585, 1.0793537184846, -0.387487727907263, NaN,
+        0.465502155580113, 3.51055687636972, 1.19157592018314, 5.5001654195619,
+        -2.9794062208591, -2.86277431544593, -2.09182559187289, -1.78589755668656,
+        -1.72005281705997, NaN, NaN, NaN, NaN, NaN, NaN, NaN, NaN,
+        NaN, NaN, NaN), dim = 100L), pcorrected = structure(c(0.000849377419153474,
+        0.0423210095424223, 0.256328046748208, 0.528586480556088,
+        0.0766221933689186, 0.129264005505931, 0.177137754498547,
+        0.105638606959378, 0.338321115867427, NaN, 0.0773809827461264,
+        0.138351665068541, 0.0846852156976564, 0.0698293923241086,
+        0.926918518829267, 9.05778875237573e-08, 0.720335604552778,
+        0.959963120931475, 0.259318212136639, NaN, 0.0954961038909552,
+        0.00148126755068367, 0.507691842184744, 0.0170784486631649,
+        0.454254318229909, 0.671140990940982, 0.0893252341570526,
+        0.840945546998515, 0.369126520991046, NaN, 0.686781943204848,
+        0.301199195429941, 0.62292953328991, 0.623518700409737, 0.16443692441598,
+        0.0134437150729616, 0.250644853277108, 0.0000808329136292585,
+        0.405017436714764, NaN, 0.0815725762954532, 0.102298995198857,
+        0.0401358267826595, 0.157642936167409, 0.0000570765610781176,
+        0.117238912014026, 0.771638738277393, 0.330828649229419,
+        0.489466621817869, NaN, 0.160878761024438, 0.190338372185203,
+        0.843470320141708, 0.0242179617346248, 0.51134932415577,
+        0.0507654829575195, 0.0736617002828646, 0.707865703158119,
+        0.0578063340496739, NaN, 0.0696059066711372, 0.382136279730124,
+        0.963740814436351, 0.614296809388373, 0.0610920797125845,
+        0.257628154136641, 0.624430468982927, 0.110368183500707,
+        0.618104574183931, NaN, 0.0000536534428476898, 0.326704874125211,
+        0.0265869880109406, 0.0000432417394942153, 0.886017825067399,
+        0.0365830352009795, 0.0654054200653444, 0.280430074299729,
+        0.698395168435894, NaN, 0.641571893614577, 0.000447169134715319,
+        0.233427571493747, 3.79435104447268e-08, 0.00288807619190024,
+        0.00419949566936639, 0.0364541201647898, 0.0741158466703972,
+        0.0854228412526985, NaN, NaN, NaN, NaN, NaN, NaN, NaN, NaN,
+        NaN, NaN, NaN), dim = 100L)), class = "data.frame", row.names = c(NA,
+        100L)), questiontypes = c("PickOne", "PickOne"),
+        footerhtml = "Age by Income&lt;br /&gt;sample size = 707; total sample size = 800; 93 missing; 95% confidence level",
+        name = "Age by Income", questions = c("Age", "Income"))
+
+
+
+test_that("Subscripting and renaming works on QStatistingTestingInfo",
+{
+    assign("ALLOW.QTABLE.CLASS", TRUE, envir = .GlobalEnv)
```

Comment:
I don't think this assignment is necessary, it only applies to the code in `flipChart` or `flipStandardCharts` I think? (forgot precisely where but it isn't in `verbs` I don't think).